cascade/auditor.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+from scipy.stats import multivariate_normal
+from ase import Atoms
+
+class BaseAuditor: 
+
+    def audit(self, atoms: list[Atoms], n_audits: int) -> tuple[float, list[Atoms]]:
+        raise NotImplementedError()
+    
+
+class ForceThresholdAuditor(BaseAuditor): 
+    """Determines the likelihood all calculations are below the threshold"""
+
+
+    def __init__(self,
+                 correction_factor : float = 1,
+                 threshold: float = 1):
+        
+        self.correction_factor = correction_factor
+        self.threshold = threshold
+
+    def audit(self, 
+              atoms: list[Atoms], 
+              n_audits: int,
+              n_sample: int = 100, 
```

Comment:
The number of sames could be an attribute of the class, passed in to init, rather than an argument here because (1) you'll probably want to use the same value for all calls and (2) it's unique to this particular implementation